sql/parser/eval.go

```diff
@@ -658,27 +658,42 @@ func transformAlias(name string) string {
 }
 
 func evalFuncExpr(expr *FuncExpr) (Datum, error) {
-	name := strings.ToLower(expr.Name.String())
-	name = transformAlias(name)
-	b, ok := builtins[name]
+	name := transformAlias(strings.ToLower(expr.Name.String()))
```

Comment:
I didn't notice this in the previous PR, but transformAlias should probably be backed by a map. Or we should add an `init()` function which copies entries for the aliases:

```
func init() {
  builtins["substr"] = builtins["substring"]
}
```